.github/workflows/php.yml

```diff
@@ -0,0 +1,21 @@
+name: PHP tests
+on: [push, pull_request]
+jobs:
+  build:
+    strategy:
+      matrix:
+        php-versions: ['5.6', '7.2']
```

Comment:
Maybe add 7.3 as it should be compatible with PrestaShop 1.7.7.0 that is itself compatible with php 7.3?

Reply:
I'm going to even see if it works for php7.4. 1.7.8 must be php7.4 compatible.